endpoints/get_address_transactions.py

```diff
@@ -60,3 +63,42 @@ async def get_transactions_for_address(
     return {
         "transactions": tx_list
     }
+
+@app.get("/addresses/{kaspaAddress}/full-transactions",
+         response_model=List[TxModel],
+         response_model_exclude_unset=True,
+         tags=["Kaspa addresses"])
+async def get_full_transactions_for_address(
+        kaspaAddress: str = Path(
+            description="Kaspa address as string e.g. "
+                        "kaspa:pzhh76qc82wzduvsrd9xh4zde9qhp0xc8rl7qu2mvl2e42uvdqt75zrcgpm00",
+            regex="^kaspa\:[a-z0-9]{61}$"),
+        limit: int = Query(
+            description="The number of records to get",
+            ge=1,
+            le=500,
+            default=50),
+        offset: int = Query(
+            description="The offset from which to get records",
+            ge=0,
+            default=0),
+        fields: str = "",
+    ):
+    """
+    Get all transactions for a given address from database.
+    And then get their related full transaction data
+    """
+
+    async with async_session() as s:
+        # Doing it this way as opposed to adding it directly in the IN clause
+        # so I can re-use the same result in tx_list, TxInput and TxOutput
+        tx_within_limit_offset = await s.execute(select(TxAddrMapping.transaction_id)
+            .filter(TxAddrMapping.address == kaspaAddress)
+            .limit(limit)
+            .offset(offset)
+            .order_by(TxAddrMapping.block_time.desc())
+        )
+
+        tx_ids_in_page = [x[0] for x in tx_within_limit_offset.all()]
+
+    return await search_for_transactions(TxSearch(transactionIds=tx_ids_in_page), fields)
```

Comment:
need to add `.order_by(Transaction.block_time.desc()))` into search_for_transactions so the ORDER is guaranteed

Reply:
Resolved in c78c378